test/recipes/70-test_sslcbcpadding.t

```diff
@@ -0,0 +1,110 @@
+#! /usr/bin/env perl
+# Copyright 2016 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the OpenSSL license (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+use strict;
+use OpenSSL::Test qw/:DEFAULT cmdstr srctop_file bldtop_dir/;
+use OpenSSL::Test::Utils;
+use TLSProxy::Proxy;
+
+my $test_name = "test_sslcbcpadding";
+setup($test_name);
+
+plan skip_all => "TLSProxy isn't usable on $^O"
+    if $^O =~ /^(VMS|MSWin32)$/;
+
+plan skip_all => "$test_name needs the dynamic engine feature enabled"
+    if disabled("engine") || disabled("dynamic-engine");
+
+plan skip_all => "$test_name needs the sock feature enabled"
+    if disabled("sock");
+
+plan skip_all => "$test_name needs TLSv1.2 enabled"
+    if disabled("tls1_2");
+
+$ENV{OPENSSL_ia32cap} = '~0x200000200000000';
```

Comment:
No idea! It's in all the other test/recipes/70-test_ssl*.t files, so I assumed it was more TLSProxy boilerplate and hoped you all would tell me if this weren't true. It seems if this line is unnecessary it is unrelated to this PR and should be fixed separately.

If I'm counting hex bits correctly, it seems to be disabling PCLMULQDQ and AESNI. Perhaps something is wrong with the ossltest engine that it can't intercept that correctly? AES-GCM stitched mode?